site/docs/v1/tech/core-concepts/applications.adoc

```diff
@@ -223,12 +223,18 @@ The issuer used by service providers (i.e. Google, Zendesk, etc.) to identify th
 [field]#Audience# [optional]#Optional#::
 Some service providers require a different audience (such as Zendesk). You can leave this blank if the audience is the same as the issuer.
 
-[field]#Callback URL (ACS)# [required]#Required#::
+[field]#Authorized redirect URLs (ACS)# [required]#Required#::
```

Comment:
I think the "ACS" should be explained int he field description, so we can remove `(ACS)` from the field name here now that have renamed it to `Authorized redirect URLs`.